conf/application.conf

```diff
@@ -61,7 +65,7 @@ braintracing {
 application {
   insertInitialData = true
   authentication {
-    enableDevAutoLogin = false
+    enableDevAutoLogin = true
```

Comment:
Why is this changed in this PR?

Reply:
@daniel-wer I noticed you changed this to false in the application.conf. was that intentional?

Reply:
No that was not intentional, but I often had to disable it to test logged-out-behavior before, so I actually preferred it being set to `false`. Logging in is easy and quick. What's your opinion @philippotto?

Reply:
I agree with @daniel-wer and would prefer it being false, too.

Reply:
fine by me :)

Reply:
:+1: